src/rules/utils.ts

```diff
@@ -1,5 +1,5 @@
 // TODO: rename to utils.ts when TS migration is complete
-import { basename } from 'path';
+import path from 'path';
```

Comment:
```suggestion
import { parse as parsePath } from 'path';
```

or just `import * as path from 'path';`

Reply:
Ok, I applied this change.